google-cloud-storage/src/main/java/com/google/cloud/storage/Blob.java

```diff
@@ -543,6 +543,13 @@ public boolean delete(BlobSourceOption... options) {
    * Blob copiedBlob = copyWriter.getResult();
    * }</pre>
    *
+   * <p>Example of moving a blob to a different bucket with a different name.
+   * <pre> {@code
+   * CopyWriter copyWriter = srcBlob.copyTo(destBucket, destBlob);
+   * Blob copiedBlob = copyWriter.getResult();
+   * boolean deleted = srcBlob.delete();
+   * }</pre>
+   *
```

Comment:
To add snippets to the javadoc use the script in `utilities/add_snippets_to_file.py` (it will check that all the tags are ther and fail otherwise).
In this case from the root of the project do:
```
python ./utilities/add_snippets_to_file.py google-cloud-examples/src/main/java/com/google/cloud/examples/storage/snippets/BlobSnippets.java google-cloud-storage/src/main/java/com/google/cloud/storage/Blob.java
python ./utilities/add_snippets_to_file.py google-cloud-examples/src/main/java/com/google/cloud/examples/storage/snippets/StorageSnippets.java google-cloud-storage/src/main/java/com/google/cloud/storage/Storage.java
```